### ISSUE TITLE
Toggle pace to speed 

### ISSUE THREAD

**Issue by dblock** [dblock/slack-strava#43]
That would be more useful for biking.

**Comment by dblock** [comment]
I added displaying both pace and speed in https://github.com/dblock/slack-strava/commit/d1b0fecd7f7990fc14bd98b99310d0c2b9a64693. I wonder whether we should choose which one depending on the type of activity though?

**Comment by dblock** [comment]
@imjared Yes, I would make the change in https://github.com/dblock/slack-strava/blob/master/slack-strava/models/activity_methods.rb#L305. Want to try to contribute?

**Comment by imjared** [comment]
For sure, thanks for pointing me to it. Probably not going to be a quick turnaround but I'll shoot a PR your way before too long.

**Comment by dblock** [comment]
Implemented in #248. When using Default fields, the bot now automatically shows the most relevant metric by activity type:

- **Pace** (min/km or min/mi) for Run, Walk, Hike, Swim, and other foot/snow activities
- **Speed** (km/h or mph) for Ride, EBikeRide, VirtualRide, Kayaking, Rowing, Canoeing, InlineSkate, RollerSki, Kitesurf, Windsurf, Surfing, StandUpPaddling

Explicitly configured fields (e.g. `set fields pace` or `set fields speed`) always override this behavior.
